README.Rmd

```diff
@@ -73,6 +88,55 @@ shinyApp(ui, server)
 knitr::include_graphics("man/figures/hierplane_spacyr.png")
 ```
 
+If you are familiar with `data.tree`, you can use it to visualize trees as well. Here is an example of generating a hierplane from YAML:
+
+```r
+library(data.tree)
+library(yaml)
+
+"
+name: r4fun
+tyler:
+  name: Tyler
+  job: Data Scientist
+  species: Human
+  toulouse:
+    name: Toulouse
+    job: Systems Engineer
+    species: Cat
+    jojo:
+      name: Jojo
+      job: Python Programmer
+      species: Dog
```

Comment:
i think you should put jojo in the `hp_datatree` example too. Right now there are two toulouses and may be confusing to some new users.

Reply:
Thanks! I'll make the change.